src/gradle/orb.yml

```diff
@@ -0,0 +1,40 @@
+executor:
+  gradle:
+    description: The docker container to use when running Gradle builds
+    docker:
+      - image: 8-jdk-node
+    environment:
+      # Disable fancy terminal updates
+      TERM: dumb
+
+commands:
+  with_cache:
```

Comment:
Consider calling this `with_gradle_cache` given how specific it is to build.gradle

Reply:
The orb is already called `gradle`, so the user will be typing `gradle/with_cache`:

Reply:
+1 to marc's naming